pymc3/tests/test_distributions.py

```diff
@@ -597,6 +597,35 @@ def check_selfconsistency_discrete_logcdf(
                 err_msg=str(pt),
             )
 
+        # Test that values below domain evaluate to -np.inf
+        if np.isfinite(domain.lower):
+            below_domain = domain.lower - 1
+            assert_equal(
+                dist.logcdf(below_domain).tag.test_value,
+                -np.inf,
+                err_msg=str(below_domain),
+            )
+
+        # Test that values above domain evaluate to 0
+        if np.isfinite(domain.upper):
+            above_domain = domain.upper + 1
+            assert_equal(
+                dist.logcdf(above_domain).tag.test_value,
+                0,
+                err_msg=str(above_domain),
+            )
+
+        # TODO: Test that logcdf wih invalid parameters is always evaluated to -inf
```

Comment:
Should this comment refer to an issue or PR? It's hard to have the context with the current line

Reply:
Forgot to remove that one!